code/modules/cm_preds/yaut_bracers.dm

```diff
@@ -95,7 +95,13 @@
 	//Non-Yautja have a chance to get stunned with each power drain
 	if(!HAS_TRAIT(M, TRAIT_YAUTJA_TECH) && !M.hunter_data.thralled)
 		if(prob(15))
+			if(istype(src, /obj/item/clothing/gloves/yautja/hunter))
```

Comment:
I may be reading this wrong but should this be set up via overriding in the child type?

Reply:
I have to do it like this to execute it before the shock_user() or they will drop the gloves in shock_user() and it won't work

also don't want to dupe code

Reply:
nvm can do better